internal/condition/eval/eval.go

```diff
@@ -25,56 +26,69 @@ func EvaluateTupleCondition(
 	tupleKey *openfgav1.TupleKey,
 	typesys *typesystem.TypeSystem,
 	context *structpb.Struct,
-) (*condition.EvaluationResult, error) {
+) (bool, error) {
```

Comment:
as discussed offline, should we undo this change?

Reply:
Yes. Please see changes.